### ISSUE TITLE
Swapping between move modes

### ISSUE THREAD

**Issue by MicroChasm** [nsmryan/RustRoguelike#339]
After the switch to swapping between move modes using shift and ctrl, it has become difficult to know which state you are in. It should be that, if you are crouching and press shift, the player stands up, and the Player part of the UI switches to saying "Running."

**Comment by nsmryan** [comment]
Is this a purely visual change, similar to showing your position if you use an item? 
The problem with actually standing up is that you could see over walls without taking a turn.

**Comment by MicroChasm** [comment]
You are right... I'm not quite sure how to handle this. I do think that it is currently confusing. The player could just stand up visually. The new position could be shown using a ghost kind of image. Maybe we could indicate the change in some other way? In the UI it could show Crouching ---> Standing or something.

**Comment by nsmryan** [comment]
I pushed a change that displays the next turns move type.
